python/hawkey/nevra-py.c

```diff
@@ -220,7 +220,7 @@ evr_cmp(_NevraObject *self, PyObject *args)
 static PyObject *
 has_just_name(_NevraObject *self, PyObject *unused)
 {
-    return hy_nevra_has_just_name(self->nevra) ? Py_True : Py_False;
+    return hy_nevra_has_just_name(self->nevra) ? PyLong_FromLong(1) : Py_False;
```

Comment:
Good work. You found problem.
But your fix is not correct. Better is something like this:
```
if (hy_nevra_has_just_name(self->nevra))
    Py_RETURN_TRUE;
else
    Py_RETURN_FALSE;

```